lib-intro.tex

```diff
@@ -72,3 +72,124 @@
 If the semantic requirements of a declaration are not satisfied at the
 point of use, the program is ill-formed, no diagnostic required.
 }
+
+\setcounter{section}{5}
+\setcounter{subsection}{2}
+\setcounter{subsubsection}{1}
+\setcounter{paragraph}{4}
+
+{\color{newclr}
+\rSec4[customization.point.object]{Customization Point Objects}
+\pnum
+A \techterm{customization point object} is an object with a
+literal class type which interacts with user-defined types while
+enforcing semantic requirements on that interaction.
+
+\pnum
+If \tcode{T} is the type of a customization point object, \tcode{T}
+shall satisfy \tcode{Semiregular<T>()} and
+\tcode{Function<const T, Args..>()} when the types of \tcode{Args..}
```

Comment:
"Each customization point object type constrains the type of its results to satisfy a particular concept."?